ocdskingfishercolab/__init__.py

```diff
@@ -247,17 +193,6 @@ def save_dataframe_to_spreadsheet(dataframe, name):
     print('Uploaded file with ID {!r}'.format(drive_file['id']))
 
 
-def download_dataframe_as_csv(dataframe, filename):
```

Comment:
I think we should keep this function, as ipython-sql doesn't offer a csv download method for DataFrames (only for result sets) and I anticipate that analysts will mostly set the `autopandas` config to `True` in order to get a nicer presentation of results.

Reply:
Makes sense, I've added it back in. Conveniently it doesn't need any changes for our new approach.